lib/routes/cdirt/namespace.ts

```diff
@@ -0,0 +1,10 @@
+import type { Namespace } from '@/types';
```

Comment:
The site does not have `cdirt` in its second level domain from the URL. Do not [create a namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) that does not follow its second level domain.